backend/schemas.py

```
@@ -51,6 +51,17 @@ class SpotCreate(BaseModel):
     crowd_level: Optional[CrowdLevel] = CrowdLevel.MEDIUM
     rating: Optional[int] = 3
 
+
+class SpotUpdate(BaseModel):
+    """スポット更新用モデル（部分更新も許容）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)